tools/witx/src/docs/ast.rs

```diff
@@ -195,8 +204,44 @@ impl ToMarkdown for StructDatatype {
 
 impl ToMarkdown for UnionDatatype {
     fn generate(&self, node: MdNodeRef) {
-        let heading = heading_from_node(&node, 1);
-        node.new_child(MdSection::new(heading, "Union variants"));
+        // Sizes & Alignments
+        let sizes_heading = heading_from_node(&node, 1);
+        node.new_child(MdSection::new(sizes_heading, "Union Layout"));
+        let union_layout = &self.union_layout();
+        node.new_child(MdNamedType::new(
```

Comment:
If you used a generic `MdSection` instead of an `MdNamedType`, then it won’t be HTML tagged by default.